source/atomics.tex

```diff
@@ -1806,6 +1988,13 @@
     @\placeholdernc{integral}@ fetch_xor(@\placeholdernc{integral}@, memory_order = memory_order::seq_cst) volatile noexcept;
     @\placeholdernc{integral}@ fetch_xor(@\placeholdernc{integral}@, memory_order = memory_order::seq_cst) noexcept;
 
+    void wait(integral, memory_order = memory_order::seq_cst) const volatile noexcept;
+    void wait(integral, memory_order = memory_order::seq_cst) const noexcept;
+    void notify_one() volatile noexcept;
+    void notify_one() noexcept;
+    void notify_all() volatile noexcept;
+    void notify_all() noexcept;
```

Comment:
No, the paper wanted them after compare_exchange_strong, but I moved them to after fetch_blah, because the latter is still changing values (similar to compare/exchange), but wait/notify don't actively change values.

Reply:
thanks - please move all of them to after the fetch_* functions, in both their synopses' declarations and itemdecls/itemdescrs.